pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -0,0 +1,315 @@
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.9.13
+ARG MAMBA_VERSION=22.9.0-1
+ARG OPEN_MPI_VERSION=4.1.1
```

Comment:
What is the reason to use 4.1.1 instead of the latest stable version (4.1.4)?

Reply:
didn't catch the newer patch.. updated.